### ISSUE TITLE
Animal Pound building has bad placement feature.

### ISSUE THREAD

**Issue by AncientSion** [CleverRaven/Cataclysm-DDA#38000]
the animal pound has a vehicle, however the vehicle is placed at the back and hence not necesarrily connected to an open street.

Picture 

# Screenshots

![2020-02-13 20_50_24-Cataclysm_ Dark Days Ahead - 0 D-12062-ge7271b0](https://user-images.githubusercontent.com/4209179/74472598-7b635f80-4ea2-11ea-8cdb-6c5e540f42e0.png)

**Comment by I-am-Erk** [comment]
I guess that vehicle got.... 

 Impounded?

**Comment by curstwist** [comment]
best solution is to remove the vehicle spawn entirely and I'm not sure anyone wants that to happen.

**Comment by I-am-Erk** [comment]
We can probably remap the building to have a driveway back there when we get around to updating it

**Comment by AncientSion** [comment]
> 
> 
> best solution is to remove the vehicle spawn entirely and I'm not sure anyone wants that to happen.

What kind of logic is that. The best way to fix it is to make the vehicle placement at the front where the tile is connected to a street. If that is not possible for some reason (gameplay ease), then rework the building so be more slim and allow a small street at the north or south border of the location tile.

**Comment by I-am-Erk** [comment]
Those are entire rewrites of the map, Sion. You're welcome to do it, but it's not a "just so this!" kind of solution, it's a large amount of work.

**Comment by curstwist** [comment]
sure, here's my logic:
1. There isn't enough space to shift the building down and fit in a 5 tile wide vehicle.  So, you wouldn't do a minor revision but a full rewrite.
2. Cars don't traditionally spawn with city mapgen.  I'm pretty sure this was added to the livestock cargo space spawned _somewhere_, which isn't an issue anymore.
3. All the monster spawns and loot spawns are in x,y coordinates so they all need to be adjusted as well (and getting animals to spawn in those tiny cages is a bit of a challenge).

The ideal compromise here is to remove it from this mapgen, make an new, different animal pound and plan on putting the vehicle in the front from the start.

I've made a beginner mapgen guide if anyone wants to pursue that option.
